nilearn/_utils/numpy_conversions.py

```diff
@@ -3,11 +3,10 @@
 # Author: Gael Varoquaux, Alexandre Abraham, Philippe Gervais
 
 import csv
+from pathlib import Path
 
 import numpy as np
 
-from .helpers import stringify_path
-
 
 def _asarray(arr, dtype=None, order=None):
     # np.asarray does not take "K" and "A" orders in version 1.3.0
```

Comment:
```suggestion
    # np.asarray does not take "K" and "A" orders in version 1.3.0
```

unrelated: seeing makes me think that we can probably simplify this function as it seems to handle behavior for old versions of numpy we do not support anymore

probably worth checking in another issue / PR